assignment4.txt

```diff
@@ -0,0 +1,9 @@
+Aly Milich
+
+1. A string is collection of characters that can make up words or sentences. In order to read a string in C, strings have to be split into character arrays which break apart the string into individual chars.
+
+2. Arrays are advantageous because they give the programmer complete control over what's inside and organizes data. It is easy and efficient to create an array and add/delete things from the array. A disadvantage is that they can be difficult to navigate sometimes and require a lot of loops to organize/search through.
```

Comment:
right - they also don't allow you to use elements of different types and their sizes can't be modified.